.github/workflows/ci.yml

```diff
@@ -101,9 +101,9 @@ jobs:
       - verify-changes
     # Don't run this job for docs/ui only PRs
     if: |
-      always() && (needs.verify-changes.result == 'skipped' ||
+      (always() && needs.verify-changes.result == 'skipped') ||
```

Comment:
this change is needed to correctly cancel go tests when a new commit is pushed to a branch